tests/baselines/reference/classMemberInitializerWithLamdaScoping4.errors.txt

```diff
@@ -14,6 +14,6 @@ tests/cases/compiler/classMemberInitializerWithLamdaScoping3_1.ts(8,21): error T
         messageHandler = () => {
             console.log(field1); // Should be error that couldnt find symbol field1
                         ~~~~~~
-!!! error TS2663: Cannot find name 'field1'. Did you mean the instance member 'this.field1'?
+!!! error TS2301: Initializer of instance member variable 'messageHandler' cannot reference identifier 'field1' declared in the constructor.

```

Comment:
This seems like a less-good error message here; similar to the `x` example, this name wouldn't be in scope, but differs in that it can be fixed by writing `this.field`, which is what the old error said.

Reply:
Fixed in the latest commit